docs/3.x/spa-tracking.md

```diff
@@ -129,6 +129,17 @@ window.addEventListener('hashchange', function() {
     _paq.push(['enableLinkTracking']);
 });
 ```
+
+## Using Matomo Tag Manager to track Single Page Application
+
+If you're using [Tag Manager](https://matomo.org/tag-manager/) to implement your Matomo Analytics Tracking, then in your Single Page Application,  whenever there is a new page loaded in your app you will need your [Matomo Tag](https://matomo.org/docs/tag-manager/#configuring-a-tag) to be triggered for the Page view to be tracked. 
+
+To trigger your Matomo tag (which calls `trackPageView`), you can either:
+1. use the "History change" [trigger](https://matomo.org/docs/tag-manager/#triggers) which would work in most cases,
+2. or in your Single Page App, you can trigger a Tag Manager Event `{event: 'mtm.PageView'}` by calling the following line in JavaScript: `_mtm.push({'event': 'mtm.PageView'});`
```

Comment:
Should we add something like `If you are using the PageView trigger do trigger a Pageview`. They could also be using DomReady or PageLoaded event etc?